oak-auth-external/pom.xml

```diff
@@ -187,7 +197,27 @@
         </dependency>
         <dependency>
             <groupId>org.apache.sling</groupId>
-            <artifactId>org.apache.sling.testing.osgi-mock</artifactId>
+            <artifactId>org.apache.sling.testing.osgi-mock.junit4</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
junit4 depends on core, therefore you only reference the former in the dependency usually in this case (https://sling.apache.org/documentation/development/osgi-mock.html#maven-dependency)

Reply:
Again a used undeclared dependency, which I usually declare explicitly as generally recommended. In this particular case, it's admittedly unnecessary.